emsize.py

```diff
@@ -37,32 +37,47 @@ def error(text):
 
 
 def parse_args(argv):
-  parser = argparse.ArgumentParser(description=__doc__)
-  parser.add_argument('-format', '--format')
-  parser.add_argument('file')
-  args = parser.parse_args(argv)
-  return args.file
+  files = []
+  params = []
+  appendFormat = True
+  ret = 0
+  for arg in argv:
+    if (arg.startswith('-')):
+      params.append(arg)
+    else:
+      files.append(arg)
+    if (arg.startswith('--format')):
+      appendFormat = False
+  if appendFormat:
+    params.append('--format=sysv')
+  for file in files:
+    ret = print_sizes(file, params)
+  return ret
+
+
+def print_sizes(file, params):
+  if not os.path.isfile(file):
+    return error('Input file %s not foune' % file)
+  ftype, fencoding = mimetypes.guess_type(file)
+  if (ftype == 'application/wasm' or ftype == 'text/vnd.a'):
+    js_size = 0
+    wasm_file = file
+  else:
+    if '.' in file:
+      file, extension = file.rsplit('.', 1)
```

Comment:
You can just do `file, extension = os.path.splitext(file)` I think.  (On need for the if check either).

Reply:
If you use the unpack syntax you need the if. Alternative it would be:
>> a, b = "test123".rsplit(".", 1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: not enough values to unpack (expected 2, got 1)

Alternative would be to not unpack and use list instead:
file = file.rsplit('.', 1)[0]